contracts/ECRecovery.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.4.11;
+
+
+/**
+ * @title Eliptic curve signature operations
+ *
+ * @dev Based on https://gist.github.com/axic/5b33912c6f61ae6fd96d6c4a47afde6d
+ */
+
+library ECRecovery {
+
+  /**
+   * @dev Recover signer address from a message by using his signature
+   * @param hash bytes32 message hash from which the signature will be recovered
```

Comment:
Maybe this isn't very clear. I just meant that the description of the param is confusing, because it sounds like the signature will be recovered instead of the address.